Doc/library/bisect.rst

```diff
@@ -21,21 +21,27 @@ example of the algorithm (the boundary conditions are already right!).
 The following functions are provided:
 
 
-.. function:: bisect_left(a, x, lo=0, hi=len(a))
+.. function:: bisect_left(a, x, lo=0, hi=len(a), key=None)
 
    Locate the insertion point for *x* in *a* to maintain sorted order.
    The parameters *lo* and *hi* may be used to specify a subset of the list
-   which should be considered; by default the entire list is used.  If *x* is
-   already present in *a*, the insertion point will be before (to the left of)
-   any existing entries.  The return value is suitable for use as the first
-   parameter to ``list.insert()`` assuming that *a* is already sorted.
+   which should be considered; by default the entire list is used. The
+   parameter *key* specifies a function of one argument that is used to
```

Comment:
Just a an opinion: what about: "*key* specifies a one argument function that.."

Reply:
Your version looks better but I'm not a native speaker and so I took inspiration `sorted()` documentation: https://docs.python.org/3/library/functions.html#sorted

I don't know which one is correct.